src/controllers/albumController.js

```diff
@@ -35,37 +34,75 @@ const updateAlbum = async (req, res) => {
   res.status(StatusCodes.OK).json({ album });
 };
 
+//will be user to let admin update price of several albums on the frontend
+
+const updatePriceOfAlbums = async (req, res) => {
+  const bulkUpdateOps = req.body.map((update) => ({
+    updateOne: {
+      filter: { _id: update.id },
+      update: { price: update.price },
+    },
+  }));
+
+  const ids = req.body.map((update) => update.id); //Creates an array of _id values from the albums to be updated in the request body.
+
+  const bulkWriteResponse = await Album.bulkWrite(bulkUpdateOps); //he response object of bulkWrite () contains information about the number of modified documents.
+  logger.info(`${bulkWriteResponse.modifiedCount} Albums updated successfully`);
```

Comment:
Do we need to `require` logger for this to work or does it just work already?

Reply:
@akosasante oh yes, when I run npm run lint:fix it gives me  50:3  error  'logger' is not defined  no-undef so I installed 'winston' library to log this messages, will commit it now